x-pack/plugins/uptime/server/rest_api/synthetics_service/add_monitor.ts

```diff
@@ -45,7 +45,7 @@ export const addSyntheticsMonitorRoute: UMRestApiRouteFactory = () => ({
       },
     ]);
 
-    if (errors) {
+    if (errors && errors.length > 0) {
```

Comment:
Discussed offline that this is causing some errors with the toasts. Let's confirm that this is resolved before merging.